.npmignore

```diff
@@ -0,0 +1,3 @@
+**/**/.certs
```

Comment:
Why not just ignore the entire tests folder? That doesn't really need to be in the `npm` package.

Reply:
Updated the PR to not copy the test folder over